src/view/Menu.svelte

```diff
@@ -1,34 +1,16 @@
 <script>
-    import List from "../components/List.svelte";
     import {viewStore} from "../stores/stores";
     import {ABOUT_VIEW, BALANCE_VIEW, RECEIVE_VIEW, SEND_VIEW, SETUP_VIEW} from "../constants/views";
-    let menu = [
-        {
-            primaryLanguageId: "receive",
-            onClick: () => viewStore.set(RECEIVE_VIEW),
-        },
-        {
-            primaryLanguageId: "send",
-            onClick: () => viewStore.set(SEND_VIEW),
-        },
-        {
-            primaryLanguageId: "balance",
-            onClick: () => viewStore.set(BALANCE_VIEW),
-        },
-        {
-            primaryLanguageId: "transactionHistory",
-        },
-        {
-            primaryLanguageId: "about",
-            onClick: () => viewStore.set(ABOUT_VIEW),
-        },
-        {
-            primaryLanguageId: "setup",
-            onClick: () => viewStore.set(SETUP_VIEW),
-        }
-    ];
+    import List from "../components/List.svelte";
+    import Primary from "../components/list/Primary.svelte";
 </script>
 
 <div id="view__menu">
-    <List elements={menu}/>
+    <List>
+        <Primary primaryLanguageId="receive" primaryText="receive" on:click={() => viewStore.set(RECEIVE_VIEW)}/>
+        <Primary primaryLanguageId="send" primaryText="send" on:click={() => viewStore.set(SEND_VIEW)}/>
+        <Primary primaryLanguageId="balance" primaryText="balance" on:click={() => viewStore.set(BALANCE_VIEW)}/>
+        <Primary primaryLanguageId="about" primaryText="about" on:click={() => viewStore.set(ABOUT_VIEW)}/>
+        <Primary primaryLanguageId="setup" primaryText="setup" on:click={() => viewStore.set(SETUP_VIEW)}/>
```

Comment:
So, `primaryText="setup"` is the fallback here?

Reply:
Yes, correct. :+1: